docs/topics/feed-exports.rst

```diff
@@ -211,38 +215,61 @@ Settings
 
 These are the settings used for configuring the feed exports:
 
- * :setting:`FEED_URI` (mandatory)
- * :setting:`FEED_FORMAT`
+ * :setting:`FEEDS` (mandatory)
+ * :setting:`FEED_EXPORT_ENCODING`
+ * :setting:`FEED_STORE_EMPTY`
+ * :setting:`FEED_EXPORT_FIELDS`
+ * :setting:`FEED_EXPORT_INDENT`
  * :setting:`FEED_STORAGES`
  * :setting:`FEED_STORAGE_FTP_ACTIVE`
  * :setting:`FEED_STORAGE_S3_ACL`
  * :setting:`FEED_EXPORTERS`
- * :setting:`FEED_STORE_EMPTY`
- * :setting:`FEED_EXPORT_ENCODING`
- * :setting:`FEED_EXPORT_FIELDS`
- * :setting:`FEED_EXPORT_INDENT`
 
 .. currentmodule:: scrapy.extensions.feedexport
 
-.. setting:: FEED_URI
+.. setting:: FEEDS
 
-FEED_URI
---------
+FEEDS
+-----
 
-Default: ``None``
+.. versionadded:: 1.7
 
-The URI of the export feed. See :ref:`topics-feed-storage-backends` for
-supported URI schemes.
+Default: ``{}``
 
-This setting is required for enabling the feed exports.
+A dictionary in which every key is a feed URI and each value is a nested
+dictionary containing configuration parameters for the specific feed.
+For instance::
 
-.. setting:: FEED_FORMAT
+    {
+        'items.json': {
+            'FORMAT': 'json',
+            'ENCODING': 'utf8',
+            'STORE_EMPTY': False,
+            'FIELDS': None,
+            'INDENT': 4,
+        }, 
+        'items.xml': {
+            'FORMAT': 'xml',
+            'FIELDS': ['name', 'price'],
+            'ENCODING': 'latin1',
+            'INDENT': 8,
+        },
+        'items.csv': {
+            'FORMAT': 'csv',
+            'FIELDS': ['price', 'name'],
+        },
+    }
 
-FEED_FORMAT
------------
 
-The serialization format to be used for the feed. See
-:ref:`topics-feed-format` for possible values.
+Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam posuere iaculis mollis. Duis cursus rutrum congue. Duis ut dolor sagittis, efficitur mi in, facilisis augue. Vivamus gravida pharetra velit id bibendum. Interdum et malesuada fames ac ante ipsum primis in faucibus.
+
+.. note::
+    
+    The ``FEED_URI`` and ``FEED_FORMAT`` settings have been deprecated in favor of :setting:`FEEDS`.
```

Comment:
@Gallaecio I know we talked about removing and non documenting deprecated stuff, but as a user I'd expect at least one line, just saying the settings I've been relying in for so long are gone.

Reply:
I personally prefer to keep that line in the release notes. With that, and as long as the old, now-deprecated settings continue to work for a while, and issue a warning about their deprecation, I think we are giving users a good upgrade path.

@kmike?

Reply:
Fair point. If we were to keep these in the main docs we'd need to monitor it and remove after a while, something that's not needed with the release notes. I'll remove it.